### PR TITLE
Making UI Intuitive to Use

### LINKED ISSUE #119: [ADMIN]: Make navigation to other user's view work properly
**Describe the bug**
Currently, when you impersonate a user it swaps the email in redux but nothing happens 

There's a series of re-render problems that I need to fix.

### LINKED ISSUE #122: [ADMIN] Fix Admin UI view
Make admin view more intuitive to use.
- when viewing a user -> the ui change is not intuitive. Should show that we are in admin view but are viewing another user. Instead of displaying "you are not an admin"
- make navigating to admin and out of admin easy

### PR DESCRIPTION
**Pull request by Zjjc123**
Made UI Intuitive to Use
- Instead of checking user email with `signedInstate.userEmail` we use `firebase.auth().currentUser?.email` to make certain UI understand when an admin is still an admin
- push `/table` to history every time we press "view"
- removing myself from admin list since I need to test non-admin user + I don't have access in firebase anyways
- change home button to dynamically redirect depend on whether the user is signed in, not signed in, or an admin

should close #119 and close #122

### CODE REVIEW

**Comment by Zjjc123** [comment]
@daniel-sudz I think this is mergeable for now to use. We can add more changes as we go. Just want to push out this admin quick before the Key Club Board transitions.  But we can continue to maintain it until we find a sustainable solution.

File: website/src/Components/Header/pageHeader.tsx [hunk 1]
```diff
@@ -74,18 +75,31 @@ const HeaderButtons: React.FunctionComponent<{}> = () => {
   );
 };
 
-const PageHeader = () => {
+const LogoButton: React.FunctionComponent<{}> = () => {
+  // make the home button redirect to admin if admin / user profile if it is a user
   const signedInstate = useSelector(selectSignedInState);
+  let url = "/";
+  if (useSelector(selectIsAdminState))
```
Comment (daniel-sudz) [inline website/src/Components/Header/pageHeader.tsx:82]: replace the if statement with the variable you defined above. You can't use hooks this way

**Comment by daniel-sudz** [comment]
Thing we didn't fix from before: you need to clear the table redux state so we don't get duplicate hour counts/entires

**Comment by daniel-sudz** [comment]
Also: slightly stylistic but there's not much point in defining a separate redux state for the isAdmin, just merge it with the current signed in-state slice to avoid the extra boilerplate.

**Comment by Zjjc123** [comment]
> Thing we didn't fix from before: you need to clear the table redux state so we don't get duplicate hour counts/entires

Shouldn't this be fixed in the Admin Branch. This PR is only fixing UI issues due to admin state. 

But yeah I will look to combine the states into one slice.

**Review by daniel-sudz — APPROVED** [review]
lgtm -- didn't test though

**Pull request merged**